src/app/app.module.ts

```diff
@@ -51,7 +52,7 @@ Amplify.configure(AWS_CONFIG);
       provide: APP_INITIALIZER,
       useFactory: initializeApplication,
       multi: true,
-      deps: [AuthService]
+      deps: [AuthService, Router]
```

Comment:
just for my own understanding is the Router added here so that amplify can do the redirect?